### DISCUSSION TITLE
A few suggestions

### DISCUSSION THREAD

**Discussion by redstonemaster60** [Leather128/LeathersFunkinEngine#226] [Ideas]
1. Add in some basic modchart events (move note x, move note y, maybe effects like shaky arrows?)
2. Maybe note types could have a few lines be used for something like playing an animation or doing something else? I know it can be done with modcharts, but if you have a modchart already you have to replace that one with the one that does it. 
Not sure if these would be difficult to add, but if possible I think they would bring a lot to the engine. Its fine if you don't want to though.

**Comment by Leather128** [comment]
1. You can do that yourself, it's not that difficult really.
2. Yeah the note types system need to be reworked anyways to provide more control over things.

**Reply by redstonemaster60** [reply]
alright thanks, also what are the playstate variables that control the x and y of the strum lines? i checked playstate.hx but there's a lot of different things when i look for it and i can't find which of them is the right one

**Reply by Leather128** [reply]
> alright thanks, also what are the playstate variables that control the x and y of the strum lines? i checked playstate.hx but there's a lot of different things when i look for it and i can't find which of them is the right one

no variables, you just change the strum x and y of strums to change them

like this:

```lua
-- sets first strum on opponent side to x 0
setActorX(0, 0)

-- sets first strum on player side to y 0
setActorY(0, 4)
```

**Reply by redstonemaster60** [reply]
alr thanks
